gdkx11/sys/Cargo.toml

```diff
@@ -60,6 +64,6 @@ tempfile = "3"
 [features]
 v3_16 = []
 v3_22 = ["v3_16"]
-v3_24 = ["v3_22"]
+v3_24 = ["v3_16"]
```

Comment:
This is wrong, see https://github.com/gtk-rs/gir/issues/979

Reply:
The never ending story...

Reply:
That broke at some point in the past and for the last update we manually changed it back here. Do the same for now unless you feel like working on gir now :) It would seem better to first get this out of the way than to include more unrelated issues

Reply:
That's what I think too. I'd like to resume work on the original changes that motivated this PR.